src/nanopyx/core/transform/sr_radial_gradient_convergence.pxd

```diff
@@ -1,6 +1,9 @@
-# autogenerated by pyx2pxd - https://github.com/HenriquesLab/pyx2pxd
-
+cdef extern from "_c_sr_radial_gradient_convergence.h":
+    double _c_calculate_dw(double distance, double tSS) nogil
+    double _c_calculate_dk(double Gx, double Gy, double dx, double dy, double distance, double tSS) nogil
```

Comment:
remove the last double tSS, its incorrect given:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/763757/222929601-0416fd88-cc9b-4522-a37e-9417868068a7.png">